docs/en/cue/cross-namespace-resource.md

```diff
@@ -0,0 +1,45 @@
+# Define resources located in defferent namespace with application
+
+In this section, we will introduce how to use cue template create resources 
+
+## Usage
+
+
+```yaml
+apiVersion: core.oam.dev/v1alpha2
+kind: WorkloadDefinition
```

Comment:
let's use componentdefinition as example

Reply:
Got it, will update docs before finish this pr